run_surefire_azure.sh

```diff
@@ -94,7 +94,7 @@ if [[ $ret != 0 ]]; then
 fi
 
 # echo "================Setting up maven-surefire"
-bash $dir/setup-custom-maven.sh "${RESULTSDIR}" "$dir" "$fullTestName" "$modifiedslug_with_sha" "$module"
+bash $dir/setup-custom-maven-tri.sh "${RESULTSDIR}" "$dir" "$fullTestName" "$modifiedslug_with_sha" "$module"
```

Comment:
One problem with the current scripts in azure-tools is that they all largely assume that they are being run on fresh VM. E.g., the contents of setup-custom-maven-tri.sh should probably check whether the custom maven-surefire is already cloned, is it on the latest version, if so then do nothing, otherwise install again and setup the extension

Eventually, perhaps all scripts should be updated to handle the case of being run multiple times on one machine. In the meantime though you should at least update  setup-custom-maven-tri.sh so that we can run some of the ISSRE experiments on one machine multiple times without the scripts doing many unnecessary actions

Reply:
Okay, I updated the setup-custom-maven-tri.sh and we can add a task to our docs to refactor the other scripts